src/code/sortable.js

```diff
@@ -105,23 +106,31 @@ export default class Sortable {
           cell.style.display = "none";
         }
       }
+      instantValues.push(data);
       return data;
     });
 
     // resolve all rows at once
     // XXX convert to allSettled and check for individual errors rather than failing hard if any row fails
     // console.log(promises);
-    this._done = Promise.all(promises).then(
-      (values) => {
-        this._items = values;
-        this.sort();
-        return true;
-      },
-      (reject) => {
-        console.log("rejected", reject);
-        this._done = false;
-      }
-    );
+    if (instantValues.length < promises.length) {
```

Comment:
It's a style thing, but I'd invert the logic of this 

if (IinstantValues.length == promises.length) {
 ...
 return
 }
 
 ...
 
 and my comment from back in the day says I wanted to try allSettled, did you look into that?

Reply:
> It's a style thing, but I'd invert the logic of this

fine

> and my comment from back in the day says I wanted to try allSettled, did you look into that?

no, while it's not fail safe, the point of failure are to be defined first (include how to settle them)
atm, if one row fails, the issue is not only the rendering of the row but login/perm/team